packages/modelence/src/client/session.ts

```diff
@@ -81,3 +83,12 @@ export function useSession() {
   const user = useSessionStore((state) => state.user);
   return { user };
 }
+
+export function isAdmin(user: User | null): boolean {
+  return user?.roles?.includes('admin') ?? false;
+}
+
+export function useIsAdmin(): boolean {
+  const user = useSessionStore((state) => state.user);
+  return isAdmin(user);
```

Comment:
Same for the frontend - can we add a `hasRole` method on the user object instead of adding yet another `useIsAdmin` hook? In general, I think we need to avoid adding a separate React hook for specific checks like this, especially if the user itself already has a session hook that includes everything we need.